src/content/guides/partial-payment-extension.mdx

```diff
@@ -39,14 +48,14 @@ When [creating a Payment Request](https://docs.centrapay.com/api/payment-request
 you can opt into partial payment by following these steps:
 
 - Set `partialAllowed` to true.
-- Set `value.amount` to the amount payable via Centrapay.
-- Set `basketAmount` to the total amount of the transaction, including external
-forms of payment (e.g., cash or card). This value should be equal to `value.amount`
-if the customer is completing their entire transaction using Centrapay.
+- Set `value.amount` to the amount payable through Centrapay.
+- Set `basketAmount` to the purchase total, including other forms of payment
+(e.g., cash or card). If the customer intends to pay the entire outstanding
```

Comment:
I don't think this sentence makes sense. Does outstanding amount mean the remaining amount?

Reply:
I have changed it to total purchase to match the value.amount